packages/react/src/Hero/Hero.tsx

```diff
@@ -15,17 +15,26 @@ type Action = {
 
 export type HeroProps = BaseProps<HTMLDivElement> & {
   heading: string | React.ReactElement
+  headingLevel?: 'h1' | 'h2' | 'h3'
```

Comment:
AFAIK the hero pattern should only ever use H1 and appear at the top of the page. We have other components like SectionIntro and CTABanner, which allow configurability for the other heading levels.

Is the configurability of the heading level super important right now? If not, wondering if we could defer this until the component rewrite has taken place. That way we can do this by simply forwarding the `as` in Heading.